playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/provider/MockBillingWrapper.java

```diff
@@ -111,7 +111,7 @@ public void triggerConsume(int responseCode, String token) {
     }
 
     public void triggerOnPurchasesUpdated() {
-        mListener.onPurchaseFlowComplete(toResult(BillingClient.BillingResponseCode.OK));
+        mListener.onPurchaseFlowComplete(toResult(BillingClient.BillingResponseCode.OK), "");
```

Comment:
Is the empty "" token a temporary placeholder?

Reply:
This is just test code, so I was going to use "" until we have some tests that actually use the value.